makederiv

```diff
@@ -565,7 +565,10 @@ while [[ "${@}" != "" ]] ; do
     fi
     if [[ "${AUDIOSTREAMCOUNT}" -gt 0 ]] ; then
         _get_audio_index "${SOURCEFILE}"
-        if [[ "${OUTPUT_TYPE}" == "broadcast" ]] ; then
+        AUDIO_STTS_ENTRIES=$(mediaconch -mt ${SOURCEFILE} | xmlstarlet sel -N mt="https://mediaarea.net/mediatrace" -v -t -m "mt:MediaTrace/mt:media/mt:block[@name='File header']/mt:block[@name='Track'][mt:block[@name='Media']/mt:block[@name='Media Information']/mt:block[@name='Sound Media Header']]/mt:block[@name='Media']/mt:block[@name='Media Information']/mt:block[@name='Sample Table']/mt:block[@name='Time to Sample']" -v "mt:data[@name='Number of entries']" -n)
+        if [[ "${AUDIO_STTS_ENTRIES}" > 2 ]] ; then
+            _add_audio_filter "aresample=async=1"
+        elif [[ "${OUTPUT_TYPE}" == "broadcast" ]] ; then
```

Comment:
ah, the conditional for AUDIO_STTS_ENTRIES isn't related to the conditional test for OUTPUT_TYPE. It should just be independent, such as

```
if [[ "${AUDIO_STTS_ENTRIES}" > 2 ]] ; then
            _add_audio_filter "aresample=async=1"
fi
```